transformations/GermanGenderSwap/test.json

```diff
@@ -0,0 +1,50 @@
+{
+  "type": "GermanGenderSwap",
+  "test_cases": [
+    {
+      "class": "GermanGenderSwap",
+      "inputs": {
+        "sentence": "Mein Vater und dein Bruder sind hier?"
+      },
+      "outputs": [{
+        "sentence": "Meine Mutter und deine Schwester sind hier?"
+      }]
+    },
+    {
+      "class": "GermanGenderSwap",
+      "inputs": {
+        "sentence": "Er ist ein Arzt und mein Vater."
+      },
+      "outputs": [{
+        "sentence": "Sie ist eine Arztin und meine Mutter."
+      }]
+    },
+    {
+      "class": "GermanGenderSwap",
+      "inputs": {
+        "sentence": "Ich sehe den Mann."
+      },
+      "outputs": [{
+        "sentence": "Ich sehe die Frau."
+      }]
+    },
+    {
+      "class": "GermanGenderSwap",
+      "inputs": {
+        "sentence": "Er ist der Prasident, sie ist die Herrscherin und sie werden jetzt bald Ehemann und Ehefrau sein!"
+      },
+      "outputs": [{
+        "sentence": "Sie ist die Prasidentin, sie ist die Herrscherin und sie werden jetzt bald Frau und Ehefrau sein!"
+      }]
+    },
+    {
+      "class": "GermanGenderSwap",
+      "inputs": {
+        "sentence": "Ich sehe, dass der Dichter und die Schauspielerin jetzt Freunde sind!"
+      },
+      "outputs": [{
+        "sentence": "Ich sehe, dass die Dichterin und die Schauspielerin jetzt Freunde sind!"
+      }]
+    }
```

Comment:
Maybe add a test case for a complex sentence from a news paper or so to show the limits.

Reply:
I've added a test case from a newspaper in the first example.